src/constants/task.ts

```diff
@@ -111,23 +129,49 @@ export const taskStatusesFilterTreeData: TreeDataEntry[] = [
     value: TaskStatus.SetupFailed,
     key: TaskStatus.SetupFailed,
   },
-  {
-    title: "Aborted",
-    value: TaskStatus.Aborted,
-    key: TaskStatus.Aborted,
-  },
-  {
-    title: "Blocked",
-    value: TaskStatus.Blocked,
-    key: TaskStatus.Blocked,
-  },
   {
     title: "Won't Run",
     value: TaskStatus.Inactive,
     key: TaskStatus.Inactive,
   },
 ];
 
+export const mapTaskStatusToUmbrellaStatus: {
+  [key: string]: string;
+} = taskStatuses.reduce(
+  (accum, { value, children }) => ({
+    ...accum,
+    ...(children
+      ? children.reduce(
+          (cAccum, child) => ({ ...cAccum, [child.value]: value }),
+          {}
+        )
+      : { [value]: value }),
+  }),
+  {}
+);
```

Comment:
This is pretty difficult to read and understand. The mix of the nested reduce's and ternaries is difficult to understand. Could we make this a little more verbose to make it easier to understand whats going on.

Reply:
Sure, I just added some comments